src/babel/transformation/transformers/minification/remove-react-prop-types.js

```diff
@@ -0,0 +1,14 @@
+import * as t from "../../../types";
+
+export var metadata = {
+  optional: true,
+  group: "builtin-setup"
+};
+
+export var Property = {
+  exit(node) {
+    if (node.key.name === 'propTypes') {
```

Comment:
See the [plugin docs](http://babeljs.io/docs/usage/plugins), notably the [visiting](http://babeljs.io/docs/usage/plugins/visiting/) ones. It's available as the second argument to a visitor function.